src/trigger.py

```diff
@@ -28,14 +28,17 @@ class Trigger(Service):
     def __init__(self, configs, args):
         super().__init__(configs, args, 'trigger')
         self._build_configs = configs['build_configs']
+        self._whoami = self._api.whoami()
```

Comment:
How about renaming the variable holding the current user's information?
```suggestion
        self._current_user = self._api.whoami()
```

Reply:
Changed